deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -304,75 +304,146 @@ spec:
+    namespace-scoped control plane and the managed resources will never converge
+    on the desired state. For more information, use this
+    [link](https://github.com/openshift/federation-dev/blob/master/docs/kubefed-scope.md)
+    
 
-        curl -Ls https://github.com/kubernetes-sigs/kubefed/releases/download/v0.1.0-rc3/kubefedctl-0.1.0-rc3-linux-amd64.tgz | tar xz
+    * A KubeFed control plane does not need to be installed on each cluster that
+    it will manage. Clusters whose resources are intended to be managed by
+    KubeFed need to be registered with the managing control plane, and the
+    control plane will interact with those clusters directly via their API.
 
+    * The instructions in this guide refer to the namespace where KubeFed is
+    installed as `<namespace>` in command examples. The upstream user guide uses
+    `kube-federation-system` as the name of this namespace.
+    
+    ### Download binaries
+
+    Once the KubeFed control plane is up and running, a user can download the
+    `kubefedctl` command-line tool to communicate with it. You can get the
+    latest release on
+    [GitHub](https://github.com/kubernetes-sigs/kubefed/releases).
+
+    ```
+     $ curl -Ls https://github.com/kubernetes-sigs/kubefed/releases/download/v0.1.0-rc5/kubefedctl-0.1.0-rc5-linux-amd64.tgz
+     $ tar -xvzf kubefedctl-0.1.0-rc5-linux-amd64.tgz
+     $ chmod u+x kubefedctl
+     $ sudo mv kubefedctl /usr/local/bin/ # make sure the location is in the PATH
+    ```
     ### Joining Clusters
 
-    Use the `kubefedctl join` command to connect clusters you want federation to
-    spread resources to. `kubefedctl join` reads information about how to connect
-    to the joining cluster and the cluster hosting federation from your
-    KUBECONFIG.
-
-        kubefedctl join cluster-name \
-          --cluster-context mycluster \            # name of a KUBECONFIG context for the cluster to join
-          --host-cluster-context mycluster \       # name of a KUBECONFIG context for the hosting cluster
-          --kubefed-namespace=<namespace> \        # namespace where federation is deployed
-          -v 2
+    Use the `kubefedctl join` command to connect clusters you want KubeFed to manage resources in. The `join` operation reads information 
+    about how to connect to the joining cluster and the cluster hosting federation from your local KUBECONFIG.
+
+    ```
+    $ kubefedctl join <cluster-name> \
+        --cluster-context mycluster \            # name of a KUBECONFIG context for the cluster to join (required when the context name differs from the cluster name)
+        --host-cluster-context mycluster \       # name of a KUBECONFIG context for the hosting cluster (required when the current-context isn't appropriate.)
+        --kubefed-namespace=<namespace> \        # namespace where federation is deployed
+        -v 2
+    ``` 
+    **Note**:     
+    * If you're using an openshift cluster for deploying the kubefed-operator then
```

Comment:
Isn't it assumed that it's being deployed in an OCP cluster?

Reply:
The description in this CSV(v0.1.0) is common for the community/upstream operator so wanted to explicitly talk about an OCP cluster.